Makefile

```diff
@@ -4,9 +4,9 @@ IDIR = include
 ODIR = obj
 
 INSTALL = install
-PREFIX = /usr/bin
```

Comment:
You need to rebase after the other PR has been merged since these changes are not part of this PR.